clang/runtime/dpct-rt/include/dpct/blas_gemm_utils.hpp

```diff
@@ -62,11 +62,11 @@ class descriptor {
 
 class matrix_layout_t {
 public:
-  enum class attribute { type, order, rows, cols, ld };
+  enum class attribute { type, order, rows, cols, ld, batch_count};
 
   matrix_layout_t(library_data_t type, std::uint64_t rows, std::uint64_t cols,
-                  std::int64_t ld)
-      : _type(type), _rows(rows), _cols(cols), _ld(ld) {}
+                  std::int64_t ld, std::uint64_t batch_count=1)
+      : _type(type), _rows(rows), _cols(cols), _ld(ld), _batch_count(batch_count) {}
```

Comment:
I prefer to keep the constructor since its parameters can 1:1 mapping to `cublasLtMatrixLayoutCreate`. We can use set/get_attribute() to access the value.
And please add a private `std::uint64_t _batch_count;` member in this class.
```suggestion
                  std::int64_t ld)
      : _type(type), _rows(rows), _cols(cols), _ld(ld)) {}
```